packages/wrangler/src/workflows/commands/instances/list.ts

```diff
@@ -88,8 +88,16 @@ export const workflowsInstancesListCommand = createCommand({
 		const prettierInstances = instances
 			.sort((a, b) =>
 				args.reverse
-					? a.created_on.localeCompare(b.created_on)
-					: b.created_on.localeCompare(a.created_on)
+					? a.created_on > b.created_on
```

Comment:
My bad here, there is no non locale aware compare on string so I think the former version is ok.